apps/web/src/views/Home/components/Banners/PerpetualSeasonalBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { FlexGap, Text, useMatchBreakpoints } from '@pancakeswap/uikit'
+import Image from 'next/legacy/image'
+import {
+  BackgroundGraphic,
+  BannerActionContainer,
+  BannerContainer,
+  BannerGraphics,
+  BannerMain,
+  BannerTitle,
+  BannerDesc,
+  FloatingGraphic,
+  GraphicDetail,
+  LinkExternalAction,
+  PancakeSwapBadge,
+  VerticalDivider,
+} from '@pancakeswap/widgets-internal'
+import { ASSET_CDN } from 'config/constants/endpoints'
+import styled from 'styled-components'
+import { useTheme } from '@pancakeswap/hooks'
+import { getPerpetualUrl } from 'utils/getPerpetualUrl'
+import { useMemo } from 'react'
+
+const bgMobile = `${ASSET_CDN}/web/banners/perpetual-season-banner/bunny-bg-mobile.png`
+const bgDesktop = `${ASSET_CDN}/web/banners/perpetual-season-banner/bunny-bg.png`
+const floatingAsset = `${ASSET_CDN}/web/banners/perpetual-season-banner/floating.png`
+const arbLogo = `${ASSET_CDN}/web/banners/perpetual-season-banner/arbSimplifiedLogo.png`
+const apxLogo = `${ASSET_CDN}/web/banners/perpetual-season-banner/apolloXCampaignLogo.png`
```

Comment:
```suggestion
const apxLogo = `${ASSET_CDN}/web/banners/perpetual-season-banner/ApolloXCampaignLogo.png`
```

Reply:
and let's unify the naming convention next time. Better keep the same pattern. Prefer kebab case for asset filenames